src/InstrumentationEngine/InstrumentationEngine.vcxproj

```diff
@@ -37,6 +45,9 @@
   <PropertyGroup Condition="'$(Platform)'=='x64'">
     <TargetName>MicrosoftInstrumentationEngine_x64</TargetName>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Platform)'=='ARM64'">
+    <TargetName>MicrosoftInstrumentationEngine_ARM64</TargetName>
```

Comment:
```suggestion
    <TargetName>MicrosoftInstrumentationEngine_arm64</TargetName>
```

Reply:
In the long term, we should consolidate the naming so that the architecture is not in the name of the assembly/shared library. No one else does this differentiation. Same with the nuget packages; we should just have one headers package and one impl package with all of the architectures inside them.